docs/application/vscode-ext/web.md

```diff
@@ -79,232 +52,236 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
+
+    > **Note**
+    >
+    > Ensure the following:
+    > - Emulator is launched, or device is connected to the PC
+    > - Chrome browser is installed
 
 4. Connect **Debug Tizen App** with **Inspector**.
-5. Open **Command Palette** and enter **Run Inspector**.
-6. Enter the port number on your local PC. For example, 9999
+5. Open **Command Palette** and type **Run Inspector**.
+6. Enter the port number to be used on the local computer. For example, 9999.
 7. In Chrome browser, the **Inspector** automatically launches and associates with the launched app.
+![Debugging you application](media/tizen_web17.png)
 
-    ![Debugging you application](media/tizen_web17.png)
+    > **Note** 
+    > 
+    > If you have a Tizen device and want to deploy the application into it, then you must first generate a certificate and add it.
 
 ### Debug Application in Web Simulator
 
 To debug application, follow these steps:
 
-1. Open **Command Palette** and enter **Run Web Simulator**.
-2. After the Web Simulator is launched, enter the path of the file in the box. For example, file:///C:/workspace/basic/index.html.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Run Web Simulator** and press **Enter**.
+3. After the **Web Simulator** is launched, enter the path of the file in the box. For example, file:///C://Workspace/basic/index.html.
 
     ![Debugging you application in the Web Simulator](media/tizen_web18.png)
 
 ## Tizen Web with Code IntelliSense
 
-VS Code Extension for Tizen Web supports IntelliSense, which includes the following features:
-
-- Code Completion
-- Hover
-- Signature Help
-- Document View document
-
-You can use these features to easily develop application using TAU.
+VS Code Extension for Tizen Web supports intelliSense. You can use these features to easily develop application using TAU.
 
 ### Features
 
-### Code completion
-  
-  Display a list of recommended values when specifying attributes or APIs.
+- Code Completion
+
+  Displays a list of recommended values when specifying attributes or APIs.
 
   ![Code completion](media/tizen_web20.gif)
 
 - Hover
   
-  Display a description of the attribute or API when the mouse over attritubes
+  Displays a description of an attribute or an API when you hover the mouse over that attribute or API.
 
-    ![Hover](media/tizen_web21.gif)
+  ![Hover](media/tizen_web21.gif)
 
 - Signature Help
   
-  Displays an information of API parameters when using the API
+  Displays information about the API parameters.
 
-    ![Signature Help](media/tizen_web22.gif)
+  ![Signature Help](media/tizen_web22.gif)
 
 - Document Viewer
   
   Displays a document related with attributes or APIs.
-    > **How to use the Document Viewer**
-    > 
-    > Click the "Search in TAU document" on the context menu
 
-    ![Document Viewer](media/tizen_web23.gif)
+  > To use the document viewer, follow these steps:
```

Comment:
Please remove the Note Tag ">". Also, please unbold  **TAU element**

Displays a document related with attributes or APIs. To use the document viewer, follow these steps:
1. Place the mouse cursor on the TAU element that you want to check and right-click.
2. Click **Search in TAU document**.

Reply:
done.